homework/schedule/schedule.hpp

```diff
@@ -0,0 +1,17 @@
+#include <chrono>
+#include <utility>
+
+template <typename Fun, typename... Args>
+void schedule(Fun&& fun,  std::chrono::duration<double> time, Args&&...args) {
+
+        auto startTimer = std::chrono::system_clock::now();
+        auto elapsed_time = [startTimer]() {
+                std::chrono::duration<double> diffrence = std::chrono::system_clock::now() - startTimer;
```

Comment:
Typo: there should be a "difference". 
Consider returning the result directly:
    return std::chrono::system_clock::now() - startTimer;
The duration can be compared directly (its compare result of count() in operator ==).